taggit/locale/eo/LC_MESSAGES/django.po

```diff
@@ -2,66 +2,69 @@ msgid ""
 msgstr ""
 "Project-Id-Version: django-taggit\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2010-09-07 09:26-0700\n"
+"POT-Creation-Date: 2019-02-06 09:31+0300\n"
 "PO-Revision-Date: 2014-03-29 18:57+0100\n"
 "Last-Translator: Baptiste Darthenay <taggit.batisteo@recursor.net>\n"
 "Language-Team: Esperanto <taggit.batisteo@recursor.net>\n"
+"Language: eo\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"Language: eo\n"
 "X-Generator: Poedit 1.5.4\n"
 
-#: forms.py:20
+#: apps.py:7
+msgid "Taggit"
+msgstr "Etikedoj"
+
+#: forms.py:27
 msgid "Please provide a comma-separated list of tags."
 msgstr "Bonvolu enmeti liston da etikedoj apartitaj per komoj."
 
-#: managers.py:39 managers.py:83 models.py:50
+#: managers.py:402 models.py:81
 msgid "Tags"
 msgstr "Etikedoj"
 
-#: managers.py:84
+#: managers.py:403
 msgid "A comma-separated list of tags."
 msgstr "Listo da etikedoj apartitaj per komoj."
 
-#: models.py:10
+#: models.py:20
 msgid "Name"
 msgstr "Nomo"
 
-#: models.py:11
+#: models.py:21
 msgid "Slug"
 msgstr "Ĵetonvorto"
 
-#: models.py:49
+#: models.py:80
 msgid "Tag"
 msgstr "Etikedo"
 
-#: models.py:56
+#: models.py:88
 #, python-format
 msgid "%(object)s tagged with %(tag)s"
 msgstr "%(object)s etikedita %(tag)s"
 
-#: models.py:100
-msgid "Object id"
-msgstr "Objekto ID"
-
-#: models.py:104 models.py:110
+#: models.py:133
 msgid "Content type"
 msgstr "Enhavtipo"
 
-#: models.py:138
+#: models.py:160 models.py:167
+msgid "Object id"
+msgstr "Objekto ID"
+
+#: models.py:175
 msgid "Tagged Item"
 msgstr "Etikedita elemento"
 
-#: models.py:139
+#: models.py:176
 msgid "Tagged Items"
 msgstr "Etikeditaj elementoj"
 
-#: contrib/suggest/models.py:57
-msgid ""
-"Enter a valid Regular Expression. To make it case-insensitive include \"(?"
-"i)\" in your expression."
-msgstr ""
-"Entajpu validan regulan esprimon. Por ke estu usklecoblinda, enmetu \"(?i)\" "
-"en via esprimo."
+#~ msgid ""
+#~ "Enter a valid Regular Expression. To make it case-insensitive include \"(?"
+#~ "i)\" in your expression."
+#~ msgstr ""
+#~ "Entajpu validan regulan esprimon. Por ke estu usklecoblinda, enmetu \"(?"
+#~ "i)\" en via esprimo."
```

Comment:
What is the reason this is commented out?

Reply:
Ah, I see. The string was removed in 8e7d741ab0a031d51aabe5ab15716e2a1920affd. :+1: